addons/website/models/ir_module_module.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class Module(models.Model):
+    _inherit = 'ir.module.module'
+
+    @api.multi
+    def _check(self):
```

Comment:
Where does this method call?

Reply:
it is called here https://github.com/odoo/odoo/blob/7115ac953777c9e5eef1e0e3e4f81e001d84b420/odoo/modules/loading.py#L217
when a module is updated

Reply:
The call to `super()` is missing.

Reply:
Indeed, done